variables.tf

```diff
@@ -42,3 +42,9 @@ variable "xchain_indexer_docker_image" {
   type        = string
   default     = "omniops/xchain-indexer:latest"
 }
+
+variable "staging_xchain_indexer_omni_config_rpc_addr" {
```

Comment:
We do not have a convention for variable naming at the moment? i just copied the style from previous vars that have target env in the name.

Reply:
correct, no convention. following other examples is 👍 

though not sure we need this to be a top level variable. we can probably just set it explicitly in `main.tf`

```
omni_config = {
  rpc_addr =  "http://staging.omni.network:8545"
}
```

Reply:
Yeah agreed here – for variables that won't change like this one, we can just set them directly in `main.tf`. For example, this is used here: https://github.com/omni-network/blockscout-terraform/blob/6a8e9f11a2ba87abdeb872c1b837c995708f2e05/main.tf#L29C10-L29C10

Since it's shared, you can create a local variable at the top of `main.tf` with something like:
```
locals {
  omni_staging_rpc = "http://staging.omni.network:8545"
}
```
and then just use `local.omni_staging_rpc`

Reply:
Fixed